.changeset/silent-tools-exist.md

```diff
@@ -0,0 +1,5 @@
+---
+'@itwin/itwinui-css': minor
```

Comment:
I kept it as minor since `pointer-events` is kind of a big change. It's also going to be in a minor release anyway, because we have other minor changes pending release.